### PR TITLE
Fix changelog widget

### LINKED ISSUE #1266: Fix header in changelog-widget
Header should be static at the top when scrolling

bilde

### PR DESCRIPTION
**Pull request by alexaoh**
Fixes #1266

Also fixes margin-bottom on changelog-widget. I think the widget looks much better now.

### CODE REVIEW

**Comment by alexaoh** [comment]
Referencing issue #1085

File: app/Resources/views/widgets/changelog_widget.html.twig [hunk 1]
```diff
@@ -24,6 +23,7 @@
                         </div>
                     </div>
                 </div>
-        {% endfor %}
+            {% endfor %}
+        </div>
     </div>
 </div>
```
Comment (AmiAhm) [inline app/Resources/views/widgets/changelog_widget.html.twig]: Hadde vært gucci om du kunne lagt på en newline på filslutt

Comment (alexaoh) [reply app/Resources/views/widgets/changelog_widget.html.twig]: Oh no, glemte det

**Comment by AmiAhm** [comment]
💯

File: app/Resources/assets/scss/control_panel.scss [hunk 2]
```diff
@@ -877,17 +877,12 @@ button.secretly-not-a-button {
...
-
-.main-card{
-  margin-bottom: 0px;
 }
 
 .changelog_card{
   border-radius: 6px;
   .dato {
-    color: grey;
+    color: #6c757d;
```
Comment (khejd) [inline app/Resources/assets/scss/control_panel.scss]: Her kan du bruke secondary _colors.scss
https://github.com/vektorprogrammet/vektorprogrammet/blob/669b472d34a1ee1e24c4d08b8e09147828311201/app/Resources/assets/scss/_colors.scss#L45

**Pull request merged**